src/trio/_core/_parking_lot.py

```diff
@@ -86,6 +88,33 @@
     from ._run import Task
 
 
+GLOBAL_PARKING_LOT_BREAKER: dict[Task, list[ParkingLot]] = {}
+
+
+def add_parking_lot_breaker(task: Task, lot: ParkingLot) -> None:
+    """Register a task as a breaker for a lot. If this task exits without being removed
+    as a breaker, the lot will break. This will cause an error to be raised for all
+    tasks currently parked in the lot, as well as any future tasks that attempt to
+    park in it.
+    """
```

Comment:
These should have a summary.

I think it might be better to link to the `break_lot` to explain the effects of a task exiting without removing themselves.

Reply:
linking to `break_lot` sounds good, which removed the need for a summary.